pex/bin/pex.py

```diff
@@ -457,9 +457,10 @@ def interpreter_from_options(options):
     return interpreter
 
 
-def build_pex(args, options, resolver_option_builder):
-  with TRACER.timed('Resolving interpreter', V=2):
-    interpreter = interpreter_from_options(options)
+def build_pex(args, options, resolver_option_builder, interpreter=None):
```

Comment:
The short answer is that I wasn't able to get that to work and I'm unsure why. :) I'm far from a pex or even python expert and the previous author of the code in question did a lot of setup work to properly initialize the interpreter. See https://github.com/twitter/heron/blob/master/3rdparty/pex/_pex.py#L135-L152

I would love to remove as much of that code as possible and simplify the integration, but I wasn't able to do so and have the build work both locally (darwin) and on CI (centos). I spent a few days trying to rip parts of that code out but was met with multiple failures that were had to diagnose. Passing the interpreter was the only way I could get the build to work.

If you're curious you can reproduce the heron build by running the command below. Or if you're up for pairing a bit on this I can revisit and we can figure out where the friction is. Or maybe it's just an easy fix that I was overlooking.
```
$ heron --config=darwin build heron/...
```